ledger/complete/ledger.go

```diff
@@ -462,25 +443,6 @@ func (l *Ledger) ExportCheckpointAt(
 
 	l.logger.Info().Msgf("checkpoint file successfully stored at: %v %v", outputDir, outputFile)
 
-	l.logger.Info().Msgf("start running post-checkpoint reporters")
-
-	if noMigration {
-		// when there is no mgiration, we generate the payloads now before
-		// running the postCheckpointReporters
-		payloads = newTrie.AllPayloads()
```

Comment:
If there is no migration, we are now skipping this time consuming call. Will save some spork time